tests/framework/extensions/clusters/clusters.go

```diff
@@ -22,6 +25,24 @@ func IsProvisioningClusterReady(event watch.Event) (ready bool, err error) {
 	return ready, nil
 }
 
+// IsHostedProvisioningClusterReady is basic check function that would be used for the wait.WatchWait func in pkg/wait.
+// This functions just waits until a hosted cluster becomes ready.
+func IsHostedProvisioningClusterReady(event watch.Event) (ready bool, err error) {
```

Comment:
wouldn't this work for any cluster

Reply:
Yes for clusters provisioned using the v3 API, I can denote that instead. Like `IsManagmentProvisioningClusterReady` if you prefer.